test/e2e/antreapolicy_test.go

```diff
@@ -814,9 +811,12 @@ func TestAntreaPolicy(t *testing.T) {
 		t.Run("Case=CNPPriorityConflictingRule", func(t *testing.T) { testCNPPriorityConflictingRule(t) })
 		t.Run("Case=CNPRulePriority", func(t *testing.T) { testCNPRulePrioirty(t) })
 		t.Run("Case=ANPBasic", func(t *testing.T) { testANPBasic(t) })
-		t.Run("Case=AuditLoggingBasic", func(t *testing.T) { testAuditLoggingBasic(t) })
 	})
-
+	// print results for reachability tests
 	printResults()
+
+	t.Run("TestGroupAuditLogging", func(t *testing.T) {
```

Comment:
@tnqn this is where the 3rd bullet point is addressed.

Reply:
This was the '4th' bullet point, I removed the 3rd after Quan's suggestion :P

Reply:
@tnqn fault for writing a relative comment :P